build.gradle.kts

```diff
@@ -62,6 +65,15 @@ subprojects {
     }
 
     kotlin { ktfmt("0.42").googleStyle() }
+
+    tasks.jacocoTestReport {
+      dependsOn(tasks.test) // tests are required to run before generating the report
+      reports {
+        xml.required.set(true)
+        csv.required.set(false)
+        html.required.set(false)
```

Comment:
Let's enable HTML reports too. I think it would be nice to be able to view a rendered report in the browser.